linkup-cli/install.sh

```diff
@@ -0,0 +1,95 @@
+if [ -x "$(command -v linkup)" ]; then
+    echo "Linkup is already installed. To update it, run 'linkup update'."
+    exit 0
+fi
+
+# region: Dependencies
+# TODO: Maybe we want this script to be able to install the dependencies as well?
```

Comment:
dnsmasq too? I don't think i needs to install them for now - but maybe it can warn without exit?

Reply:
Oof, skipped that one 😅 
Fixed on 5513a62!